src/DocumentFormat.OpenXml/Packaging/RelationshipErrorHandler.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.IO.Packaging;
+using System.Xml.Linq;
+
+namespace DocumentFormat.OpenXml.Packaging
+{
+    /// <summary>
+    /// A class that holds information about invalid relationships in <see cref="PackagePart"/> instances.
+    /// </summary>
+    public abstract class RelationshipErrorHandler
```

Comment:
I think there should be at least one default implementation in the repository. This means that there is no need for each developer to implement a specific type by himself

Reply:
Maybe. What would you suggest as the default implementation?

Reply:
@twsouthwick Can we use the `RemoveMalformedHyperlinksRelationshipErrorHandler` code?

Reply:
I'd be happy to have an implementation like that live in the project, but it is definitely only test code at the moment. We would want a way to know we are removing all hyperlinks - it is special cased for spreadsheets.

Reply:
I've got a default implementation that just takes a delegate to rewrite. See `RelationshipErrorHandler.CreateRewriterFactory`